node/subsystem-util/src/lib.rs

```diff
@@ -50,6 +50,7 @@ use streamunordered::{StreamUnordered, StreamYield};
 use thiserror::Error;
 
 pub mod validator_discovery;
+pub mod metered;
```

Comment:
since we're splitting subsystem-util in #2214, maybe we could put it in a separate crate to avoid another batch of merge conflicts?

Reply:
Sure thing!